docker/images/master/k8s/launch_test.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
We should probably move launch_test.sh and teardown.sh to a new folder, perhaps /test/, to keep the distinction between testing scripts and actual code clear.